src/std_bridge/string.swift

```diff
@@ -25,7 +25,7 @@ class SwiftString {
 }
 
 extension RustString {
-    func toString() -> String {
+    public func toString() -> String {
```

Comment:
By the way, I'm not sure which other methods should be declared public as well

Reply:
Right for sure - no worries we can handle that over time.